hassio/src/addon-store/hassio-addon-repository.ts

```diff
@@ -66,17 +67,35 @@ class HassioAddonRepositoryEl extends LitElement {
             <paper-card
               .addon=${addon}
               class=${addon.available ? "" : "not_available"}
-              @click=${this.addonTapped}
+              @click=${this._addonTapped}
             >
               <div class="card-content">
                 <hassio-card-content
                   .hass=${this.hass}
                   .title=${addon.name}
                   .description=${addon.description}
                   .available=${addon.available}
-                  .icon=${this.computeIcon(addon)}
-                  .iconTitle=${this.computeIconTitle(addon)}
-                  .iconClass=${this.computeIconClass(addon)}
+                  icon=${addon.installed !== addon.version
+                    ? "hassio:arrow-up-bold-circle"
+                    : "hassio:puzzle"}
+                  .iconTitle=${addon.installed
+                    ? addon.installed !== addon.version
+                      ? "New version available"
+                      : "Add-on is installed"
+                    : addon.available
+                    ? "Add-on is not installed"
+                    : "Add-on is not available on your system"}
+                  .iconClass=${addon.installed
+                    ? addon.installed !== addon.version
+                      ? "update"
+                      : "installed"
+                    : !addon.available
+                    ? "not_available"
+                    : ""}
+                  .iconImage=${ha105pluss && addon.icon
+                    ? `/api/hassio/addons/${addon.slug}/icon`
+                    : undefined}
+                  ?showBlueTopbar=${addon.installed}
```

Comment:
It's always faster to go via properties. Only use attributes if the value never changes. 
```suggestion
                  .showBlueTopbar=${addon.installed}
```